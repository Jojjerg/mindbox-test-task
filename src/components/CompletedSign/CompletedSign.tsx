import React from "react";
import ready from "../../assets/icons/check.png";

const CompletedSign = () => {
  return (
    <div data-testid="completed-sign">
      <img
        className="w-[30px] h-[30px]"
        src={ready}
        width={30}
        height={30}
        alt="ready"
      />
    </div>
  );
};

export default CompletedSign;
