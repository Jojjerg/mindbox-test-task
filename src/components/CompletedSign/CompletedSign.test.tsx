import { render, screen } from '@testing-library/react';

import CompletedSign from './CompletedSign';
import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

const renderer = createRenderer();

const defaultComponent = <CompletedSign />;

describe('<CompletedSign />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(defaultComponent)
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
})