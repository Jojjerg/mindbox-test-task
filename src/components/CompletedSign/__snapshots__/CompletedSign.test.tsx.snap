// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CompletedSign /> should render and match the snapshot 1`] = `
<div
  data-testid="completed-sign"
>
  <img
    alt="ready"
    className="w-[30px] h-[30px]"
    height={30}
    src="test-file-stub"
    width={30}
  />
</div>
`;
